src/index.ts

```diff
@@ -1546,6 +1546,11 @@ export class CachedMerklePatriciaTree<K, V> extends MerklePatriciaTree<K, V> {
     return this.maxCacheDepth;
   }
 
+  errors = {
+    ERR_KEY_NOT_FOUND: new Error('Get Failed: Key Mismatch'),
```

Comment:
this is a bit strange. Can you just declare a new Error type

```typescript
export class MerklePrunedError extends Error {
   constructor() {
      super("Failed: path pruned in tree and no matching nodes found in node map");
   }
}
```

also, stuff like this, in the future needs to be a static field (why would you need a per-instance version of it?)